### PR TITLE
[AAE-10668] - Fix [object Object] for License entitlements in Admin App's About page

### PR DESCRIPTION
**Pull request by DiogoABastos**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
The About page in the Admin App shows [object Object] for License entitlements
Screen Shot 2022-09-27 at 15 11 22
**What is the new behaviour?**
The About page in the Admin App shows correct object with icons for License entitlements
Screen Shot 2022-09-27 at 15 12 13

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by eromano — CHANGES_REQUESTED** [review]
Lock file should not be committed if you didn't touch any package.json dep

File: lib/core/src/lib/utils/object-utils.ts [hunk 1]
```diff
@@ -65,4 +65,59 @@ export class ObjectUtils {
...
+    static isEmpty(target: any): boolean {
+        return target && Object.keys(target).length === 0 && Object.getPrototypeOf(target) === Object.prototype;
+    }
+
+    static hasKeys(target: any): boolean {
+        return target && Object.keys(target).length > 0;
+    }
+
+    static isBooleanObject(target: any): boolean {
+        return Object.values(target).every(value => typeof value === "boolean");
```
Comment (alep85) [inline lib/core/src/lib/utils/object-utils.ts:82]: can you replace double quotes with single quotes?

Comment (DiogoABastos) [reply lib/core/src/lib/utils/object-utils.ts:82]: Sure!

File: lib/core/src/lib/utils/object-utils.ts [hunk 2]
```diff
@@ -82,7 +82,7 @@ export class ObjectUtils {
         return Object.values(target).every(value => typeof value === "boolean");
     }
 
-    static booleanPrettify(target: any): string {
+    static booleanPrettify(target: any, enhancer: any = null): string {
```
[the remark this answers is not in the document]
Comment (alep85) [reply lib/core/src/lib/utils/object-utils.ts:85]: could be `enhancer?: (param: string) => string`

File: lib/core/src/lib/utils/object-utils.ts (hunk 1 above)
Comment (andrea-ligios) [reply lib/core/src/lib/utils/object-utils.ts]: We should automate this kind of thing with ESLint though, @popovicsandras.

Formatting, line ending, whitespace trimming, and proper quotation... that should come out for free in VSCode, when leaving the line

File: lib/core/src/lib/utils/object-utils.spec.ts [hunk 3]
```diff
@@ -130,4 +130,120 @@ describe('ObjectUtils', () => {
...
+            expect(ObjectUtils.booleanPrettify(nonBooleanObjOne)).toBe(nonBooleanObjOne.toString());
+            expect(ObjectUtils.booleanPrettify(nonBooleanObjTwo)).toBe(nonBooleanObjTwo.toString());
+        });
+
+        it('should return string with either &#9989 or &#10060 symbols if object with boolean values', () => {
+            const obj = {
+                testOne: true,
+                testTwo: false
+            };
+            expect(ObjectUtils.booleanPrettify(obj)).toContain('&#9989' || '&#10060');
```
[the remark this answers is not in the document]
Comment (DiogoABastos) [reply lib/core/src/lib/utils/object-utils.spec.ts:235]: Absolutely!

**Review by eromano — APPROVED** [review]

**Review by andrea-ligios — APPROVED** [review]
LGTM!

Let's rebase and wait for a green CI, then you'll be able to merge

**Pull request merged**